superset/sqllab/command.py

```diff
@@ -110,7 +115,21 @@ def run(  # pylint: disable=too-many-statements,useless-suppression
                 "status": status,
                 "payload": self._execution_context_convertor.serialize_payload(),
             }
-        except (SqlLabException, SupersetErrorsException) as ex:
+        except SupersetErrorsException as ex:
+            if all(ex.error_type == SupersetErrorType.SYNTAX_ERROR for ex in ex.errors):
+                raise SupersetSyntaxErrorException(ex.errors) from ex
+            raise ex
+        except SupersetException as ex:
```

Comment:
I noticed that we were catching SqlLabException before. This one is a parent class, correct, so it should still work?